frontend/src/clients/BoardClient.ts

```diff
@@ -20,9 +20,8 @@ export class BoardClient extends HttpClient {
 
   getVerifyBoard = async (params: getVerifyBoardParams) => {
     try {
-      const result = await this.axiosInstance
-        .get(`/boards/${params.type}`, { params: { ...params } })
-        .then((res) => res)
+      const boardType = params.type === 'Classic Jira' ? '/boards/Classic_Jira' : `/boards/${params.type}`
```

Comment:
boardType = params.type === 'Classic Jira' ? 'Classic_Jira' : 'params.type'
await this.axiosInstance.get(`/boards/${boardType}` { params }).then((res) => res)

Reply:
right, I agree with this improvement